modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -2047,8 +2203,10 @@ Besides the default `--help`, `-f`/`--fields` and `-o`/`--output` options, this
 > runtime-mgr standalone-alert list [options]
 ----
 
-This command lists all alerts for standalone Mules in the current environment. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all alerts for standalone Mules in the current environment. 
```

Comment:
I don't think we allow "Mules." Prob should be "Mule instances".

Reply:
Yep!